com.obsproject.Studio.Plugin.RewardsTheater.yaml

```diff
@@ -0,0 +1,57 @@
+id: com.obsproject.Studio.Plugin.RewardsTheater
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/RewardsTheater
+cleanup:
+  - /include
+  - /lib/cmake
+  - '*.a'
+modules:
+  - name: boost
+    buildsystem: simple
+    build-commands:
+    - ./bootstrap.sh --prefix=/app/plugins/RewardsTheater --with-libraries=system,url,json
+    - ./b2 -j $FLATPAK_BUILDER_N_JOBS install
+    sources:
+    - type: archive
+      url: https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.bz2
+      sha256: a6e1ab9b0860e6a2881dd7b21fe9f737a095e5f33a3a874afc6a345228597ee6
+
+  - name: fmt
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=None
```

Comment:
None is useful if you don't want some default optimisation/debug flags set in the env to be not overridden. Either don't use this, let it use whatever upstream uses, or use Release/RelWithDebInfo